homeassistant/alerts.py

```diff
@@ -0,0 +1,87 @@
+"""Get alerts from 'alerts.home-assistant.io' and show as persistent notifications."""
```

Comment:
Does it make sense to make this a new integration and potentially load it in `default_config` instead?

Reply:
That was my initial thought as well, but then I leaned towards putting it into the core directly, since it felt quite integral.

Any good name ideas for the integration? `alert` is already taken.